frontend/web/components/pages/OrganisationSettingsPage.js

```diff
@@ -1248,12 +1248,14 @@ const OrganisationSettingsPage = class extends Component {
                                       <Button theme='text'>
                                         {webhook.url}
                                       </Button>
-                                      <div className='list-item-footer faint'>
-                                        Created{' '}
-                                        {moment(webhook.created_date).format(
-                                          'DD/MMM/YYYY',
-                                        )}
-                                      </div>
+                                      {webhook.created_at ? (
+                                        <div className='list-item-footer faint'>
+                                          Created{' '}
+                                          {moment(webhook.created_at).format(
+                                            'DD/MMM/YYYY',
+                                          )}
+                                        </div>
+                                      ) : null}
```

Comment:
Do we need the `: null` here?

Reply:
Q for @kyle-ssg I took the snippet from another part of the page https://github.com/Flagsmith/flagsmith/blob/6e6b9bea004b1fa523ba4aa594d5dc88a5ce6076/frontend/web/components/pages/OrganisationSettingsPage.js#L1098

Reply:
Can you just run the frontend and verify that it works?

Reply:
Yes i have!